src/components/ChallengeEditor/index.js

```diff
@@ -131,6 +134,29 @@ class ChallengeEditor extends Component {
     this.resetChallengeData(this.setState.bind(this))
   }
 
+  deleteModalLaunch () {
+    if (!this.state.isDeleteLaunch) {
+      this.setState({ isDeleteLaunch: true })
+    }
+  }
+
+  async onDeleteChallenge () {
+    const { deleteChallenge, challengeDetails, history } = this.props
+    console.log(challengeDetails, deleteChallenge)
+    try {
+      this.setState({ isSaving: true })
+      // Call action to delete the challenge
+      await deleteChallenge(challengeDetails.id)
+      this.setState({ isSaving: false })
+      this.resetModal()
+      history.push(`/projects/${challengeDetails.projectId}/challenges`)
+    } catch (e) {
+      const error = _.get(e, 'response.data.message', 'Unable to Delete the challenge')
+      console.log(error)
```

Comment:
@mark-nakachon let's remove `console.log` as I guess it's only needed for debugging.